src/Building/UnixBuild.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Php\Pie\Building;
```

Comment:
`Building` looks a bit weird, given that the initial implementation seems to be more zend/Symfony-esque, guess you didn't like having `Build` there?

Reply:
`Building` is just the domain concern here, no strong opinions on frameworks here, it just is what it is :shrug: